webapp/src/main/webapp/src/app/shared/cachingData.service.ts

```
@@ -25,7 +25,7 @@ export class CachingDataService {
     observable
       .subscribe(x => {
         this.cache[url] = x;
-      });
+      }, (err) => Observable.throw(err));
```

Something about the .subscribe not having an explicit error handler causes the error to be swallowed here...